starter/source/elements/elbuf_init/elbuf_ini.F

```diff
@@ -1993,8 +1994,8 @@ SUBROUTINE ELBUF_INI(ELBUF_TAB ,MAT_PARAM,
               NVARTMP = ELBUF_TAB(NG)%BUFLY(IL)%NVARTMP
               NUVAREOS= ELBUF_TAB(NG)%BUFLY(IL)%NVAR_EOS
               NPTT    = ELBUF_TAB(NG)%BUFLY(IL)%NPTT
-              NFAIL   = ELBUF_TAB(NG)%BUFLY(IL)%NFAIL
-              IF (NFAIL == 1) ALLOCATE(BUFLY%FAIL(NPTR,NPTS,NPTT))                
+              NFAIL   = ELBUF_TAB(NG)%BUFLY(IL)%NFAIL                                                
+              IRUPT   = MAT_PARAM(IMAT(IL))%FAIL(1)%IRUPT   ! modele de rupture     
```

Comment:
can you translate to english :-) 

Thanks,
Sébastien

Reply:
Hi @sebastienVilleneuve , 

Sorry for this, bad copy paste of a line containing french comment. Thanks a lot for the review !

Reply:
Thanks @vdavaze